packages/aws-cdk/lib/api/util/cloudformation/stack-status.ts

```diff
@@ -25,22 +25,14 @@ export class StackStatus {
     return this.name.endsWith('FAILED');
   }
 
-  get isRollback(): boolean {
-    return this.name.indexOf('ROLLBACK') !== -1;
-  }
-
-  get isStable(): boolean {
+  get isInProgress(): boolean {
     return !this.name.endsWith('_IN_PROGRESS');
```

Comment:
Be careful! The logic here needs to be reversed. This checks if the stack is not in progress.

Alternatively, rename to `isNotInProgress` but that's weird.